openupgrade_scripts/apriori.py

```diff
@@ -0,0 +1,30 @@
+""" Encode any known changes to the database here
+to help the matching process
+"""
+
+# Renamed modules is a mapping from old module name to new module name
+renamed_modules = {
+    # odoo
```

Comment:
remember to check and fill this file and redo the analysis :)

Reply:
btw check out https://github.com/OCA/OpenUpgrade/issues/4707#issuecomment-2603133987 for never again doing this manually

Reply:
Yeah, but the bot will never fill apriori automatically haha

```python
merged_modules = {
    # odoo
    "account_audit_trail": "account",
    "account_payment_term": "account",
    "l10n_br_pix": "l10n_br",
    "l10n_es_edi_facturae_adm_centers": "l10n_es_edi_facturae",
    "l10n_es_edi_facturae_invoice_period": "l10n_es_edi_facturae",
    "l10n_fr_invoice_addr": "l10n_fr_account",
    "l10n_ro_efactura": "l10n_ro_edi",
    "payment_sips": "payment_worldline",
    "sale_product_configurator": "sale",
    "website_sale_picking": "website_sale_collect",
    # OCA/...
}
```